Source/Core/Core/ActionReplay.cpp

```diff
@@ -245,13 +245,13 @@ std::vector<ARCode> LoadCodes(const IniFile& global_ini, const IniFile& local_in
           }
           else
           {
-            PanicAlertT("Action Replay Error: invalid AR code line: %s", line.c_str());
+            PanicAlertFmtT("Action Replay Error: invalid AR code line: {0}", line);
```

Comment:
Shouldn't this be `{}` to be consistent with the other strings with only one parameter.

Reply:
This is to stay consistent with the changes introduced within #9253